app/(gcforms)/[locale]/(form administration)/form-builder/[id]/responses/[statusFilter]/components/ManageFormAccessDialog/UserActions.tsx

```diff
@@ -52,7 +54,7 @@ export const UserActions = ({
           <div className="flex flex-row gap-1">
             <span>{user.expired ? t("expired") : t("invited")}</span>
             <div className="inline-block">
-              <Tooltip.Simple text={t("resend")} side="top">
+              <Tooltip.Simple text={t("resend", { email })} side="top">
```

Comment:
The idea is to create a relationship between what the button will do and which user will impacted. There are lots of ways to do this but this seems simplest. (same for below changes)